ytfzf

```diff
@@ -427,7 +435,7 @@ scrape_peertube () {
 				channel: .channel.displayName,
 				thumbs: .thumbnailUrl,
 				duration: "\(.duration / 60 | floor):\(.duration % 60)",
-				views: .views,
+				views: "\(.views)",
```

Comment:
seen this in other places also, isn't `.views` equivalent to `\(.views)`?

Reply:
"\(.views)" is a string, so it can be added to other strings, it throws an error when views is an int.